base/parse.jl

```diff
@@ -31,6 +31,12 @@ julia> parse(Int, "afc", base = 16)
 julia> parse(Float64, "1.2e-3")
 0.0012
 
+julia> parse(Float32, "24.9f-3")     # f literal throws an error
+ERROR: ArgumentError: cannot parse "24.9f-3" as Float32
+
+julia> parse(Float32, "$(24.9f-3)")  # interpolate to bypass it
+0.0249f0
+
```

Comment:
This just parses the string "0.0249". But seems unlikely to be the solution to anyone's problem, why parse if you have the number?
```suggestion
julia> parse(Float32, "$(24.9f-3)")  # interpolate to bypass it
0.0249f0

```

Reply:
Yeah, interpolating a number into a string to then parse it doesn't make much sense.

Reply:
Should this be removed before merging? I see you added the label.

Reply:
Must have added it to the wrong PR.